site/cypress/e2e/Cypress-Feature/forums.spec.js

```diff
@@ -122,12 +158,26 @@ describe('Test cases revolving around creating, replying to, merging, and removi
         // Tutorial
         replyToThread(title3, reply3);
 
-        // Upduck the first post of each thread
-        upduckPost(title1);
-        upduckPost(title2);
-        upduckPost(title3);
-
-        // Check the stats page for instructor with 3 upducks
+        // Student upduck
```

Comment:
At this point, this single test case is very bloated. I would prefer you split upducks into its own test case instead.

Reply:
I talked about it with Cameron before I did this, it is suppose to be done this way.

Reply:
Sounds good then

Reply:
Cypress tests are okay to be big, and they are meant to test a 'end to end' from the start to the end of a given section, so everything in Forums can ideally be tested in a single test (if we can iron out the flakeyness)

Reply:
However, if it runs more than 1 minute per test, i guess it's okay to start splitting them out. If they require a previous test to have run before it will pass, then those should be combined, (like create, reply, delete, etc), but if you want, you can add a new forum post to test upducks, if you think it would reduce flakeyness later.